conf/zmconfigd.log4j.properties

```diff
@@ -1,16 +1,44 @@
-log4j.threshhold=OFF
-log4j.rootLogger=INFO,ZMCONFIGDLOG
-log4j.appender.ZMCONFIGDLOG=org.apache.log4j.DailyRollingFileAppender
-log4j.appender.ZMCONFIGDLOG.File=/opt/zimbra/log/zmconfigd-log4j.log
-log4j.appender.ZMCONFIGDLOG.DatePattern='.'yyyy-MM-dd
-log4j.appender.ZMCONFIGDLOG.Append=true
-log4j.appender.ZMCONFIGDLOG.layout=org.apache.log4j.PatternLayout
-log4j.appender.ZMCONFIGDLOG.layout.ConversionPattern=%d %-5p [%t] [%x] %c{1} - %m%n
-log4j.appender.AUDIT=org.apache.log4j.rolling.RollingFileAppender
-log4j.appender.AUDIT.RollingPolicy=org.apache.log4j.rolling.TimeBasedRollingPolicy
-log4j.appender.AUDIT.RollingPolicy.FileNamePattern=/opt/zimbra/log/zmconfigd-audit.log.%d{yyyy-MM-dd}
-log4j.appender.AUDIT.File=/opt/zimbra/log/zmconfigd-audit.log
-log4j.appender.AUDIT.layout=org.apache.log4j.PatternLayout
-log4j.appender.AUDIT.layout.ConversionPattern=%d %-5p [%t] [%x] %c{1} - %m%n
-log4j.additivity.zimbra.security=false
-log4j.logger.zimbra.security=INFO,AUDIT
+status = error
+dest = err
+name = PropertiesConfig
+
+# Appender AUDIT writes to the file "audit.log".
+appender.ZMCONFIGDLOG.type = RollingFile
+appender.ZMCONFIGDLOG.name = configdFile
+appender.ZMCONFIGDLOG.fileName = /opt/zimbra/log/zmconfigd-log4j.log
+appender.ZMCONFIGDLOG.filePattern = /opt/zimbra/log/zmconfigd-log4j.log.%d{yyyy-MM-dd}
+appender.ZMCONFIGDLOG.layout.type = PatternLayout
+appender.ZMCONFIGDLOG.layout.pattern = %d %-5p [%t] [%x] %c{1} - %m%n
+appender.ZMCONFIGDLOG.policies.type = Policies
+appender.ZMCONFIGDLOG.policies.time.type = TimeBasedTriggeringPolicy
+appender.ZMCONFIGDLOG.policies.time.interval = 2
+appender.ZMCONFIGDLOG.policies.time.modulate = true
+appender.ZMCONFIGDLOG.strategy.type = DefaultRolloverStrategy
+
+
+# Appender AUDIT writes to the file "audit.log".
+appender.AUDIT.type = RollingFile
+appender.AUDIT.name = auditFile
+appender.AUDIT.fileName = /opt/zimbra/log/zmconfigd-audit.log
```

Comment:
@desouzas @jeastman @dasiyogesh I didn't find any reference which is writing logs in this file. Do you have any idea about this?

Reply:
The story: https://bugzilla.zimbra.com/show_bug.cgi?id=79564